src/main/resources/logback.groovy

```diff
@@ -29,4 +40,4 @@ logger("bio.terra.service.tabulardata.google.BigQueryProject", DEBUG);
 */
 
 // root sets the default logging level and appenders
-root(INFO, ["Console-Appender"])
+root(INFO, [System.getenv().getOrDefault("TDR_LOG_APPENDER", "Console-Stackdriver")])
```

Comment:
I wonder if we should flip the logic, i.e. by default use `Console-Standard` or `Console-Appender` and in our pods, use `Console-Stackdriver` instead by setting environment variables in the Helm charts. We do this for a number of other static environment variables. In addition to being backwards compatible with how we are currently doing logs, that way it's not an additional thing we need to manually set when debugging locally.

Reply:
I was wondering about that.  I figured the more common case would be running in pods so we'd want the JSON format as the default.  I'm curious, what's a good other example of us doing that?  I'm just thinking we may want our default more aligned to the cloud deployment but I'm happy to explicitly switch the default and set in the Helm charts

Reply:
My vote is to only set it in my local developer environment and not have to visit all of the helm charts. I already have a script that deals with setting vault tokens, authentication, and project and bucket environment variables. One more won't be a problem for me.